packages/kbn-relocate/utils/relocate.ts

```diff
@@ -92,12 +92,14 @@ export const calculateModuleTargetFolder = (module: Package): string => {
 };
 
 export const isInTargetFolder = (module: Package, log: ToolingLog): boolean => {
-  if (!module.group || module.group === 'common' || !module.visibility) {
+  const { group, visibility } = module.manifest;
+
+  if (!group || group === 'common' || !visibility) {
```

Comment:
Fix a bug in the script: use `group` and `visibility` from the manifest, rather than those inferred from _path_.